src/style/app.module.css

```
@@ -4188,3 +4189,1540 @@ button[data-testid='createPostBtn'] {
+  font-size: 1.2rem;
+  margin-bottom: 20px;
+  background: var(--grey-bg-color);
+  border-radius: 10px;
+  padding: 5px;
+}
+
+.calendar__header {
+  display: flex;
+  margin-bottom: 2rem;
+  align-items: center;
+  margin: 0px 10px 0px 10px;
```

Biome (1.9.4)

[error] Unexpected shorthand property margin after margin-bottom



(lint/suspicious/noShorthandPropertyOverrides)